docs/architecture/adr-004-application-links-expiration.md

```diff
@@ -0,0 +1,249 @@
+# ADR 004: Expiration of application links
+
+## Changelog
+
+- September 20th, 2021: Initial draft;
+- September 21th, 2021: Moved from DRAFT to PROPOSED;
+- December  22th, 2021: First review;
+- January   04th, 2022: Second review.
+
+## Status
+
+PROPOSED
+
+## Abstract
+
+Currently when a user links their centralized application with the Desmos profile, the created link contains a timestamp of when it has been created.   
+Since centralized applications' username can be switched and sold, we SHOULD implement an "expiration date" system on links. 
+This means that after a certain amount of time passes, the link will be automatically marked as invalid, and the user has to connect it again in order to keep it valid.
```

Comment:
```suggestion
Currently when a user links a centralized application with their Desmos profile, the created link contains a timestamp of when it has been created.   
Since centralized applications' username can be switched and sold, we SHOULD implement an "expiration date" system on links. This means that after a certain amount of time passes, the link will be automatically deleted and the user has to connect it again in order to keep it valid.
```